frontend/src/config/KialiIcon.tsx

```diff
@@ -118,6 +119,7 @@ export const KialiIcon: { [name: string]: React.FunctionComponent<IconProps> } =
   OutOfMesh: (props: IconProps) => conversorIconProps(props, <BlueprintIcon />),
   Pause: (props: IconProps) => conversorIconProps(props, <PauseIcon />),
   PauseCircle: (props: IconProps) => conversorIconProps(props, <PauseCircleIcon />),
+  PencilAlt: (props: IconProps) => conversorIconProps(props, <PencilAltIcon />),
```

Comment:
Maybe `Edit` has more meaning for Kiali application than `PencilAlt` (actually it is the edit icon for Kiali application that will be applied in future features)